ow_power_system/src/power_system_node.cpp

```diff
@@ -17,27 +25,117 @@ int main(int argc, char* argv[]) {
 
   //Construct our power draw listener
   //TODO: Check out latching
```

Comment:
If there is work to do please make a Jira ticket for it.

Reply:
There already is for the power listener, I'll update the description to include checking out latching.